### PR TITLE
Document relationship between "busy" status and team review assignment

### LINKED ISSUE #10112: Relationship between user status and team review assignment is not documented
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/organizations/organizing-members-into-teams/managing-code-review-assignment-for-your-team
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#setting-a-status

### What part(s) of the article would you like to see updated?

The settings page for each team contains the following note:

> Note: this won't assign members who have committed to the pull request or have set their status to Busy.

However, regular members (i.e. not `maintainer` or org `owner`) cannot see this view. The documentation never explicitly mentions the interaction between these two features.

At a minimum the code review assignment article should describe the fact that "Busy" users will not be selected for automatic assignment. It's probably worth linking back from the profile article as well.

### Additional information

PR incoming!

### PR DESCRIPTION
**Pull request by bendrucker**
### Why:

Closes https://github.com/github/docs/issues/10112

This adds information on how the user status and code review assignment features interact. It links from the documentation of each feature to the other. 

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@bendrucker 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ When you set your status, you can also let people know that you have limited ava
 
 ![Requested reviewer shows "busy" note next to username](/assets/images/help/profile/request-a-review-limited-availability-status.png)
 
-If you select the "Busy" option, when people @mention your username, assign you an issue or pull request, or request a pull request review from you, a note next to your username will show that you're busy.
+If you select the "Busy" option, when people @mention your username, assign you an issue or pull request, or request a pull request review from you, a note next to your username will show that you're busy. You will also be excluded from automatic review assignment for any pull requests assigned to any teams you belong to. For more information, see "[Managing code review assignment for your team](/organizations/organizing-members-into-teams/managing-code-review-assignment-for-your-team)."
```
Comment (lecoursen) [inline content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md:103]:
```suggestion
If you select the "Busy" option, when people @mention your username, assign you an issue or pull request, or request a pull request review from you, a note next to your username will show that you're busy. You will also be excluded from automatic review assignment for pull requests assigned to any teams you belong to. For more information, see "[Managing code review assignment for your team](/organizations/organizing-members-into-teams/managing-code-review-assignment-for-your-team)."
```

File: content/organizations/organizing-members-into-teams/managing-code-review-assignment-for-your-team.md [hunk 2]
```diff
@@ -33,6 +33,8 @@ The round robin algorithm chooses reviewers based on who's received the least re
 
 The load balance algorithm chooses reviewers based on each member's total number of recent review requests and considers the number of outstanding reviews for each member. The load balance algorithm tries to ensure that each team member reviews an equal number of pull requests in any 30 day period.
 
+Any team members that have set their status to "Busy" will not be selected for review. If all team members are busy, the pull request will remain assigned to the team itself. For more information, see "[Setting a status](/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#setting-a-status)."
```
Comment (lecoursen) [inline content/organizations/organizing-members-into-teams/managing-code-review-assignment-for-your-team.md:36]:
```suggestion
Any team members that have set their status to "Busy" will not be selected for review. If all team members are busy, the pull request will remain assigned to the team itself. For more information about user statuses, see "[Setting a status](/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#setting-a-status)."
```

**Review by lecoursen — DISMISSED** [review]
Thanks for a very high quality PR! I'm applying some teeny suggestions and then I'll merge. ⚡

**Review by lecoursen — APPROVED** [review]

**Pull request merged**